test/pages/auctionOptions.html

```diff
@@ -0,0 +1,199 @@
+<!-- This is a Test Page for configuring Auction Options. One bidder will be marked as secondary and the auction not wait for their bids to return.  -->
+<!DOCTYPE html>
```

Comment:
test pages in this repo tend to get out of date. Perhaps this better suited to the documentation site?

Reply:
That makes sense, I won't be able to host a built version of pbjs with auctionOptions included in order to show this page with jsfiddle, so I instead I will include the logging snippet to the troubleshooting page.